update.sh

```diff
@@ -1095,6 +1095,21 @@ setup_auto_update() {
     fi
 }
 
+check_free_space() {
+    dockerRootDir="$(docker info --format='{{ print .DockerRootDir }}')"
+    freeSpace="$(df --block-size=1M "${dockerRootDir}" | tail -1 | awk '{print $4}')"
```

Comment:
`df -P -m "${dockerRootDir}"` for portability (-P is POSIX_ME_HARDER, --block-size doesn't work with busybox).